static/js/utils/tile_utils.tsx

```diff
@@ -376,6 +377,18 @@ export function getSourcesJsx(sources: Set<string>): JSX.Element {
   );
 }
 
+// Processes a unit string
```

Comment:
Let's add a comment that this converts a units with "^" into superscript.

Reply:
done